services/management/ia/templates_service.go

```diff
@@ -54,11 +54,11 @@ const (
 	dirPerm            = os.FileMode(0o775)
 )
 
-// templateInfo represents alerting rule template information from various sources.
+// TemplateInfo represents alerting rule template information from various sources.
 //
 // TODO We already have models.Template, iav1beta1.Template, and alert.Template.
 //      We probably can remove that type.
-type templateInfo struct {
+type TemplateInfo struct {
```

Comment:
That type should be removed eventually. We should not move into the opposite direction by exporting it.